wagtail/admin/forms/account.py

```diff
@@ -115,7 +122,55 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._original_avatar = self.instance.avatar
 
+    @contextmanager
+    def create_avatar_context(self, avatar):
+        avatar_context = willow.Image.open(avatar)
+        yield avatar_context
+
+    def reduce_avatar(self, avatar):
+        default_avatar_size = (400, 400)
```

Comment:
Suggestion: Move this to a constant outside the function. 

Thought: Should it be configurable?

Reply:
Based on the previous conversation i had with @gasman he talked about the need of it not being configurable which is why i didn't bother https://github.com/wagtail/wagtail/pull/12623#discussion_r1866582083 . I can make it configurable though